test/util/test_parse.py

```diff
@@ -4,9 +4,12 @@
 import unittest
 from mycroft.util.parse import normalize
 from mycroft.util.parse import extractnumber
-
+from mycroft.util.parse import extractdatetime_en
```

Comment:
Go ahead and switch to using extract_datetime() (not the _en()) version once that change is made.